sharding-transaction-spi-impl/sharding-transaction-base-spi-impl/sharding-transaction-base-saga/src/main/java/io/shardingsphere/transaction/saga/utils/JDBCUtil.java

```diff
@@ -95,16 +87,12 @@ public static void executeUpdate(final Connection connection, final String sql,
      * @throws SQLException failed to execute SQL, throw this exception
      */
     public static void executeBatch(final Connection connection, final String sql, final Collection<Collection<Object>> paramsCollection) throws SQLException {
-        PreparedStatement preparedStatement = null;
-        try {
-            preparedStatement = connection.prepareStatement(sql);
+        try (PreparedStatement preparedStatement = connection.prepareStatement(sql);) {
             for (Collection<Object> params : paramsCollection) {
```

Comment:
variable name in for each loop should name as each

Reply:
ok